src/main/java/org/jruby/ext/openssl/PKeyEC.java

```diff
@@ -1059,6 +1071,61 @@ public IRubyObject inspect() {
             return ObjectSupport.inspect(this, (List) Collections.singletonList(entry));
         }
 
+        @Deprecated
+        public IRubyObject initialize(final ThreadContext context, final IRubyObject[] args) {
+            final int argc = Arity.checkArgumentCount(context.runtime, args, 1, 2);
+
+            switch (argc) {
+                case 1:
+                    return initialize(context, args[0]);
+                case 2:
+                    return initialize(context, args[0], args[1]);
+                default:
+                    throw context.runtime.newArgumentError(args.length, 1);
+            }
+        }
+
+        @JRubyMethod(name = "mul")
+        public IRubyObject mul(final ThreadContext context, final IRubyObject[] args) {
```

Comment:
I think there is at least one other math operation on `ECPoint` that this change now makes it easy to support: [add](https://docs.ruby-lang.org/en/3.3/OpenSSL/PKey/EC/Point.html#method-i-add)

Reply:
Yes I'll do add in another PR!